test/evergreen.yml

```diff
@@ -565,7 +565,32 @@ functions:
         pip3 install lxml==4.8.0 Pygments==2.11.2 Jinja2==3.0.3 gcovr==5.0
         mkdir -p ../coverage_report
         GCOV=/opt/mongodbtoolchain/v4/bin/gcov gcovr -r .. -f ../src -e '.*/bt_(debug|dump|misc|salvage|vrfy).*' -e '.*/(log|progress|verify_build|strerror|env_msg|err_file|cur_config|os_abort)\..*' -e '.*_stat\..*' -e 'bench' -e 'examples' -e 'test' -e 'ext' -e 'dist' -e 'tools' -j 4 --html-self-contained --html-details ../coverage_report/2_coverage_report.html --json-summary-pretty --json-summary ../coverage_report/1_coverage_report_summary.json
-        python3 ../test/evergreen/code_coverage_analysis.py -c ../coverage_report/1_coverage_report_summary.json -t ../time.txt
+        python3 ../test/evergreen/code_coverage_analysis.py -s ../coverage_report/1_coverage_report_summary.json -t ../time.txt
+
+        # Generate Atlas compatible format report.
+        if [ ! -z ${generate_atlas_format} ]; then
+            python3 ../test/evergreen/code_coverage_analysis.py -c component_coverage -o ../coverage_report/atlas_out_code_coverage.json -s ../coverage_report/1_coverage_report_summary.json -t ../time.txt
+        fi
+
+  "upload-component-coverage-report-atlas":
+    - command: shell.exec
+      params:
+        shell: bash
+        script: |
+          set -o errexit
+          set -o verbose
```

Comment:
should get rid of these, to avoid printing the username and password in the logs, and making silent: true.

Reply:
Thanks for pointing this out, `upload-component-coverage-report-atlas` is obsolete now so I will remove this function.

Reply:
Thanks!